scripts/blender_load_replay.py

```diff
@@ -0,0 +1,348 @@
+- Also specify `--settings-path settings.json` for dataset specific blender settings (to avoid manually changing the settings in blender for every replay).
+Example `settings` json with all supported fields:
+```
+{
+  "lights": [
+    {
+      "type": "SUN",
+      "name": "my-light",
+      "energy": 3.0,
+      "location": [1.40667, -2.66486, 6.1511]
+    }
+  ]
+}
+```
+
+"""
+
+
+try:
+    import bpy
+except ImportError:
+    raise ImportError(
+        "Failed to import Blender modules. This script can't run "
+        "standalone. Run `blender --python path/to/blender_load_replay.py ...`. Watch the terminal for "
+        "debug/error output."
+    )
+
+import argparse
+import json
+import os
+from dataclasses import dataclass
+
+
+@dataclass
+class ImportItem:
+    filepath: str
+    do_join_all: bool = True
+    force_color: list = None
+
+
+def import_scene_helper(raw_filepath):
+    """
+    Import a single asset into Blender
+    """
+
+    # These fixups will be tried one at a time, in order, until a matching, existing file is found
+    model_filepath_fixups = [
+        # use uncompressed versions when available
+        ("/stages/", "/stages_uncompressed/"),
+        ("/urdf/", "/urdf_uncompressed/"),
+        None,
+    ]
+
+    filepath = None
+    for fixup in model_filepath_fixups:
+        raw_filepath = raw_filepath.split("?")[0]
+        if fixup is None:
+            filepath = raw_filepath
+        else:
+            if raw_filepath.find(fixup[0]) != -1:
+                filepath = raw_filepath.replace(fixup[0], fixup[1])
+            else:
+                continue
+        if os.path.exists(filepath):
+            break
+    else:
+        raise RuntimeError("can't find file " + raw_filepath, filepath)
+
+    ext = os.path.splitext(filepath)[-1].lower()
+    if ext == ".dae":
+        # Convert to GLB.
+        orig_filepath = filepath
+        filepath = filepath.replace(".dae", ".glb")
+        raw_filepath = raw_filepath.replace(".dae", ".glb")
+        if not os.path.exists(filepath):
+            os.system(f"assimp export {orig_filepath} {filepath}")
+        ext = ".glb"
+
+    if ext == ".glb" or ext == ".gltf":
+        if "ycb" in filepath:
+            filepath += ".orig"
+
+        filename = os.path.basename(filepath)
+        bpy.ops.import_scene.gltf(
+            filepath=filepath, files=[{"name": filename}]
+        )
+    elif ext == ".obj":
+        bpy.ops.import_scene.obj(filepath=filepath)
+    elif ext == ".stl":
+        bpy.ops.import_mesh.stl(filepath=filepath)
+    else:
+        raise RuntimeError("no importer found for " + filepath)
+
+    return filepath
+
+
+def import_item(item):
+    import_scene_helper(item.filepath)
+
+    bpy.ops.object.select_all(action="SELECT")
+    if len(bpy.context.selected_objects) == 0:
+        raise ValueError("No objects found in scene")
+    bpy.context.view_layer.objects.active = bpy.context.selected_objects[0]
+    if "Stage" in item.filepath:
```

Comment:
Let's make this a command-line option for the script. Some people definitely will want the ceiling and they'll be super-confused that it is missing. You could even make it entirely command-line-driven (force the user to specify "Light" and "ceiling" on the command line).

Reply:
For future reference: a more robust way to cull the ceiling would be the Bisect tool, with a user-specified height. See this tutorial for how it works: https://aihabitat.org/tutorial/editing_in_blender/ . But no need to add this feature in this PR.